bylaws/officers.md

```diff
@@ -1,18 +1,54 @@
 # Officers of the Corporation
 ## Offices held
-The officers of this corporation shall be a Secretary Functionary, a Treasurer Functionary, and at least one Facilitation Functionary.
+The officers of this corporation shall be a number of Functionaries, with a minimum of a Secretary Functionary and Treasurer Functionary at all times. The corporation, at the board's discretion, may also have a chairman of the board.
 
-Any number of offices may be held by the same person, except that neither the Secretary Functionary nor the Treasurer Functionary may serve concurrently as either the President or Chairman of the Board.
+Neither the Secretary Functionary nor the Treasurer Functionary may serve concurrently as either the President or Chairman of the Board.
 
 {% include "../bylaw-pending.md"%}
-## Responsibilities
+
+## Appointment of the Chairman of the Board
+At the board's discretion, a member of the board of directors may be may be chosen annually by the board to be appointed to the office of chairman of the board and shall serve at the pleasure of the board, subject to the rights of any officer under any employment contract.
+
+{% include "../bylaw-pending.md"%}
+
+## Assumption of Office of Functionary
+As a collective, functionary positions are voluntary roles taken on by willing members and intended to change regularly. No person may hold more than one position simultaneously, unless permitted by the membership. Any member or non-member of this corporation may perform any of the functions of any of the Functionaries, however the Functionaries are expected to perform their duties regularly and must perform them if no one else can or will.
+
+{% include "../bylaw-pending.md"%}
+
+## Qualifications
+Any member or non-member of this corporation may assume the office of a Functionary.
+
+{% include "../bylaw-pending.md"%}
+
+## Removal and Replacement
+A Functionary may be replaced by another qualified person at any time during a Regular Meeting of the membership by a simple majority vote, or a resolution by the Board of Directors.
```

Comment:
Have we defined elsewhere what’s needed (e.g., simple majority) for the board of directors to make a resolution?